### DISCUSSION TITLE
Questions Regarding the Square Tool

### DISCUSSION THREAD

**Discussion by vladQll** [xournalpp/xournalpp#4176] [Q&A]
Hello I am using the latest version 1.1.1+dev in windows 10, I have used the triangular square but I do not understand how it works, if I can draw the horizontal and diagonal lines corresponding to the angle, but **No** I can drag the tool, the only way to move it is when I press the arrows (up, down, left and right) but it is not so precise when I want to place it in a specific place, Another question is, this square can be rotated? because if so, **No** I can not do it I do not know how I could rotate it I have tried with the central wheel of the mouse but it does not work

https://user-images.githubusercontent.com/28522011/182044895-f9bd70f9-349d-4ad8-add9-85a0c61a768f.mp4

.

**Comment by vladQll** [comment]
@rolandlo

**Reply by rolandlo** [reply]
Use the modifier `Alt` to move the setsquare more precisely. Press `r` (or `R`) for rotation, `s` (or `S`) for making it larger/smaller.

Do you have a touchscreen? Then you can also manipulate the setsquare by touch. If you have a Wacom tablet or the sort (touchpad input device), that won't work though.

Please take a look at the [Setsquare Readme](https://github.com/xournalpp/xournalpp/blob/master/development/documentation/Setsquare_Readme.md) for more information.

**Reply by vladQll** [reply]
oh I understand, I'll settle for the alt and r keys because I have a wacom intuos small. thank you very much
